README.md

```diff
@@ -1,174 +1,18 @@
-[![CocoaPods](https://img.shields.io/cocoapods/p/libPhoneNumber-iOS.svg?style=flat)](http://cocoapods.org/?q=libPhoneNumber-iOS)
-[![CocoaPods](https://img.shields.io/cocoapods/v/libPhoneNumber-iOS.svg?style=flat)](http://cocoapods.org/?q=libPhoneNumber-iOS)
-[![Travis](https://travis-ci.org/iziz/libPhoneNumber-iOS.svg?branch=master)](https://travis-ci.org/iziz/libPhoneNumber-iOS)
-[![Coveralls](https://coveralls.io/repos/iziz/libPhoneNumber-iOS/badge.svg?branch=master&service=github)](https://coveralls.io/github/iziz/libPhoneNumber-iOS?branch=master)
-[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
+# Signal's fork of [libPhoneNumber-iOS][]
 
-# **libPhoneNumber for iOS**
+This is Signal's fork of libPhoneNumber-iOS, which is an Objective-C port of [Google's libphonenumber][libphonenumber].
 
- - NBPhoneNumberUtil
- - NBAsYouTypeFormatter
- - NBTextField.swift (Swift 3)
+## How to update metadata from libphonenumber
 
-> ARC only, or add the **"-fobjc-arc"** flag for non-ARC
+If you want the latest metadata from libphonenumber...
 
-## Update Log
-[https://github.com/iziz/libPhoneNumber-iOS/wiki/Update-Log](https://github.com/iziz/libPhoneNumber-iOS/wiki/Update-Log)
+1. Clone this fork.
+1. Determine the version of libphonenumber (not libPhoneNumber-iOS) that you want to pull metadata from. For example, v1.2.3. You may wish to visit [the libphonenumber repo][libphonenumber].
+1. From the project root, run `LIBPHONENUMBER_REF=v1.2.3 make update_metadata`, replacing the variable with whatever version you want. This should update a few files; most notably, `libPhoneNumber/NBPhoneNumberMetaData.plist`.
```

Comment:
It feels important to me to call out that this triggers a script that actually downloads content from the main `libphonenumber` repo, and parses it to do stuff. E.g., if that repo ever gets renamed/moved etc, the scripts here will fail. Can we call that out?

Reply:
Done in 36917a4d4279ffd4989f6ac13ed3109adaa633dc.